staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/kmsv2/envelope_test.go

```diff
@@ -113,6 +127,10 @@ func TestEnvelopeCaching(t *testing.T) {
 			cacheSize:                0,
 			simulateKMSPluginFailure: false,
 		},
+		{
+			desc:      "cache disabled, plugin failure because of invalid annotation should fail",
+			cacheSize: 0,
+		},
```

Comment:
uh where did this come from?

Reply:
Ah, it was from a dirty rebase. Removed it!